nx-X11/lib/modules/im/ximcp/Makefile.am

```diff
@@ -0,0 +1,56 @@
+NULL =
+
+noinst_LTLIBRARIES=libximcp.la
+
+AM_CPPFLAGS =					\
+	-I$(top_srcdir)/include			\
+	-I$(top_srcdir)/include/X11		\
+	-I$(top_builddir)/include 		\
+	-I$(top_builddir)/include/X11		\
+	-I$(top_srcdir)/src/xcms		\
+	-I$(top_srcdir)/src/xkb			\
+	-I$(top_srcdir)/src/xlibi18n		\
+	-I$(top_srcdir)/src			\
+	-I$(top_srcdir)/../exports/include	\
+	-D_GNU_SOURCE				\
```

Comment:
> AFAIR we have to use _GNU_SOURCE, otherwise some Linux headers will fail.

Steven Chamberlain and I here at DebConf have actually manually / visually compared the different individual flags and macros being set. Unfortunately, we did not document, but the final outcome was: we dearly surely truely want _GNU_SOURCE.